_data/events.json

```diff
@@ -0,0 +1,235 @@
+[
+   {
+      "name":"AMTA 2024",
+      "id": "amta-2024",
+      "description": "16th biennial conference of the Association for Machine Translation in the Americas",
+      "start_date":"2024-09-30",
+      "end_date":"2024-10-02",
+      "location":{
+         "type":"VirtualLocation",
+         "name":"Online",
+         "event_attendance_mode": "OnlineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Organization",
+         "name":"Association for Machine Translation in the Americas",
+         "url":"https://amtaweb.org"
+      },
+      "important_dates":
+         {
+            "Submission deadline": "2024-06-06",
+            "Notification of acceptance":"2024-07-18",
+            "Final “camera-ready” papers for proceedings": "2024-09-16"
+         }
+   },
+   {
+      "name":"AmericasNLP 2024",
+      "id": "americasnlp-2024",
+      "description": "Machine translation for indigenous languages of the Americas",
+      "start_date":"2024-06-20",
+      "end_date":"2024-06-21",
+      "location":{
+         "type":"Place",
+         "name":"Mexico City, Mexico",
+         "event_attendance_mode": "OfflineEventAttendanceMode"
+      },
+      "organizer":{
+         "type":"Person",
```

Comment:
not a person

Reply:
https://iwslt.org/2023/#organizers

It looks like the organisers of this event are individual people and not an organisation/association?